<?xml version="1.0" encoding="utf-8"?>
<launch>
  
  <arg name="rob_ns" default="rob1" />
  
  <group ns="$(arg rob_ns)">
	  <param name="kuka_msg_cmd" type="string" value="getCtrl" />
	  
	  <node name="test_kuka_service" pkg="kuka_msgs" type="kuka_msgs_node" 
			respawn="false" 
			output="screen"
			args=""/>
  </group>	
</launch>
